content/en/docs/setup/production-environment/turnkey/tencent.md

```diff
@@ -0,0 +1,13 @@
+## Running Kubernetes on Tencent Kubernetes Engine
+
+### Tencent Kubernetes Engine
+
+ [Tencent Cloud Tencent Kubernetes Engine (TKE)](https://intl.cloud.tencent.com/product/tke) provides native Kubernetes container management services. You can deploy and manage a Kubernetes cluster with TKE in just a few steps. For detailed directions, see [Deploy Tencent Kubernetes Engine](https://intl.cloud.tencent.com/document/product/457/11741).
+
+ In addition to full compatibility with the native Kubernetes API,TKE features Kubernetes plugins of Tencent Cloud such as [Cloud Block Storage](https://intl.cloud.tencent.com/product/cbs) and [Cloud Load Balancer](https://intl.cloud.tencent.com/product/clb), helping you build a complete system of container services and functions. For learn more about TKE in its official [TKE documentation](https://intl.cloud.tencent.com/document/product/457).
```

Comment:
```suggestion
 In addition to full compatibility with the native Kubernetes API,TKE features Kubernetes plugins of Tencent Cloud such as [Cloud Block Storage](https://intl.cloud.tencent.com/product/cbs) and [Cloud Load Balancer](https://intl.cloud.tencent.com/product/clb), helping you build a complete system of container services and functions. To learn more, see the official [TKE documentation](https://intl.cloud.tencent.com/document/product/457).
```

@yunxji: What do you think about moving “To learn more…” into a What's Next? section.

Reply:
![image](https://user-images.githubusercontent.com/50287034/61204828-dd978e80-a720-11e9-9b47-2f550e0e988e.png)
@sftim Excuse me, is this what you mean?

Reply:
A What's Next section is like this example:
https://github.com/kubernetes/website/blame/abcee2dccd17c94376595b34cc05e56345a7f06c/content/zh/docs/concepts/workloads/pods/podpreset.md#L66

Reply:
I thought for a moment and thought that turning into A What's Next section might be a big project. Now this may be more concise, and readers can see more systematic and detailed instructions in official documents.Can there be no A What's Next section?

Reply:
@sftim Sorry, I have some misunderstandings about your suggestion before. Now I have made the following changes based on your suggestions. Please review
![image](https://user-images.githubusercontent.com/50287034/61212708-a1baf400-a735-11e9-8d06-23e0e2a66abb.png)

Reply:
@sftim 

> A What's Next section is like this example:
> https://github.com/kubernetes/website/blame/abcee2dccd17c94376595b34cc05e56345a7f06c/content/zh/docs/concepts/workloads/pods/podpreset.md#L66


Is this the case? If this is possible, please help merge this PR.

![image](https://user-images.githubusercontent.com/50287034/61291573-6dad0500-a801-11e9-8a1b-492100a60e22.png)